exchanges/bitmex/bitmex_wrapper.go

```diff
@@ -690,6 +691,16 @@ func (b *Bitmex) ValidateCredentials() error {
 }
 
 // GetHistoricCandles returns candles between a time period for a set time interval
-func (b *Bitmex) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval time.Duration) (kline.Item, error) {
+func (b *Bitmex) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
+	return kline.Item{}, common.ErrNotYetImplemented
+}
+
+// GetHistoricCandlesEx returns candles between a time period for a set time interval
+func (b *Bitmex) GetHistoricCandlesEx(pair currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
+	if !b.KlineIntervalEnabled(interval) {
```

Comment:
I'm guessing that you left this in for convenience? Otherwise can just return kline.Item{}, common.ErrNotYetImplemented (and same for the other wrappers)